paimon-web-ui-new/src/views/metadata/components/datafile/index.tsx

```diff
@@ -0,0 +1,95 @@
+/* Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+import { type DataTableColumns } from 'naive-ui'
+
+type RowData = {
+  partiTion: string
+  bucket: number
+  filePath: string
+  fileFormat: string
+}
+
+
+export default defineComponent({
+  name: 'MetadataDataFile',
+  setup() {
+    const { t } = useLocaleHooks()
+
+    const data: RowData[] = [
+      {
+        partiTion: '[1]',
+        bucket: 0,
+        filePath: 'opt/paimon/warehouse/ods.db/t_role',
+        fileFormat: 'ORC'
+      },
+      {
+        partiTion: '[2]',
+        bucket: 0,
+        filePath: 'opt/paimon/warehouse/ods.db/t_role',
+        fileFormat: 'ORC'
+      },
+      {
+        partiTion: '[3]',
+        bucket: 1,
+        filePath: 'opt/paimon/warehouse/ods.db/t_role',
+        fileFormat: 'ORC'
+      },
+      {
+        partiTion: '[4]',
+        bucket: 1,
+        filePath: 'opt/paimon/warehouse/ods.db/t_role',
+        fileFormat: 'ORC'
```

Comment:
```suggestion
        fileFormat: 'PARQUET'
```

Reply:
It is just a placeholder and will be deleted after subsequent interface connection